meta-intel-edison-bsp/conf/machine/edison.conf

```diff
@@ -22,10 +22,7 @@ KBUILD_DEFCONFIG="i386_defconfig"
 KCONFIG_MODE="--alldefconfig"
 
 KERNEL_MODULE_AUTOLOAD += " ledtrig-heartbeat ledtrig-timer tun"
-# FIXME: file parameter should be based on partition UUID (from U-Boot) or fixed
-# with label (label seems to work, but driver is probed too early)
-#module_conf_g_multi = "options g_multi file=/dev/mmcblk0p9 stall=0 idVendor=0x8087 idProduct=0x0A9E iProduct=Edison iManufacturer=Intel"
-#KERNEL_MODULE_PROBECONF += "g_multi"
 
+PREFERRED_VERSION_acpica = "20210105"
```

Comment:
We have 2 sources: the poky layer and meta-acpi. meta-acpi in the hardknott branch and poky both have 20210105.
But it's a moving target. Poky Honnister has 20210730. So meta-intel-edison honnister (in my repo) is already on that one.